Makefile

```diff
@@ -1,19 +1,24 @@
-prepare-check:
-	ansible-playbook -b -K -i inventory-local -c local ./recipe-tests.yaml
+CONTAINER_NAME=packit
+CONTAINER_EXEC=podman exec $(CONTAINER_NAME)
 
-check:
-	tox
+test_container: test_container_remove
+	rsync -a $(CURDIR)/ /tmp/packit
+	podman build --tag $(CONTAINER_NAME) .
+	podman run --name $(CONTAINER_NAME) -ti -d -v /tmp/packit:/src:Z $(CONTAINER_NAME)
+	sleep 2
+
+test_container_remove:
+	podman stop $(CONTAINER_NAME) || true
+	podman rm -f $(CONTAINER_NAME) || true
+	podman image rm $(CONTAINER_NAME) || true
 
-# build-tests: recipe-tests.yaml
-# 	ansible-bender build -- ./recipe-tests.yaml $(SOURCE_GIT_IMAGE) $(PACKIT_TESTS_IMAGE)
+install:
+	pip3 install --user .
 
-# shell:
-# 	podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(PACKIT_TESTS_IMAGE) bash
+check:
+	find . -name "*.pyc" -exec rm {} \;
+	PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --color=yes --verbose --showlocals --cov=packit --cov-report=term-missing tests
```

Comment:
prepending PYTHONPATH=$(CURDIR) should make it work without the need of `install`